sdk/storage/azure-storage-file-share/CHANGELOG.md

```diff
@@ -14,6 +14,8 @@ pointing to the root of the file share would raise an `InvalidResourceName` on a
 Python 3.12.
 - Fixed an issue where authentication errors could raise `AttributeError` instead of `ClientAuthenticationError` when
 using async OAuth credentials.
+- Fixed an issue where specifying datetime objects with less than 7 digits of precision as input could incorrectly raise
+`InvalidHeaderValue` due to improper precision parsing.
```

Comment:
Just wanted to check we are happy with this @jalauzon-msft 😄

Reply:
lgtm